aten/src/ATen/native/sparse/ValidateCompressedIndicesCommon.h

```diff
@@ -348,18 +391,41 @@ void validate_compressed_sparse_indices_kernel(
     const int64_t cdim,
     const int64_t dim,
     const int64_t nnz) {
+  constexpr size_t idx_max_ndims = 8; // up to 7-dim batch.
```

Comment:
```suggestion
  constexpr int64_t idx_max_ndims = 8; // up to 7-dim batch.
```

To avoid issues like:

```
aten/src/ATen/native/sparse/ValidateCompressedIndicesCommon.h:414:19: error: comparison of integers of different signs: 'const int64_t' (aka 'const long') and 'const size_t' (aka 'const unsigned long') [-Werror,-Wsign-compare]
[CONTEXT] [2023-03-02T09:56:19.461-08:00]     if (idx_ndims <= idx_max_ndims) {
[CONTEXT] [2023-03-02T09:56:19.461-08:00]         ~~~~~~~~~ ^  ~~~~~~~~~~~~~

```